randovania/data/json_data/prime3.txt

```diff
@@ -2067,59 +2427,86 @@ Hall of Remembrance
 Asset id: 6915612318130423578
 > Dock to Imperial Crypt (Upper); Heals? False
   * Normal Door to Imperial Crypt/Dock to Hall of Remembrance (Upper)
-  > Pickup (Ship Missile Expansion)
-      Use Screw Attack (Space Jump)
+  > Locked Door Ledge
+      Trivial
 
 > Dock to Imperial Crypt (Lower); Heals? False
   * Normal Door to Imperial Crypt/Dock to Hall of Remembrance (Lower)
+  > Dock to Tower (Lower)
+      Use Screw Attack (Space Jump)
   > Pickup (Screw Attack)
       Trivial
 
 > Dock to Tower (Lower); Heals? False
   * Normal Door to Tower/Dock to Hall of Remembrance (Upper)
-  > Pickup (Screw Attack)
+  > Dock to Imperial Crypt (Lower)
       Use Screw Attack (Space Jump)
 
 > Dock to Tower (Upper); Heals? False
   * Normal Door to Tower/Dock to Hall of Remembrance (Lower)
+  > Dock to Tower (Lower)
+      Trivial
   > Pickup (Ship Missile Expansion)
       Use Screw Attack (Space Jump)
+  > Locked Door Ledge
+      Use Screw Attack (Space Jump)
 
 > Pickup (Screw Attack); Heals? False
   * Pickup 48; Major Location? True
   > Dock to Imperial Crypt (Lower)
       Trivial
```

Comment:
Change this connection to "Event - Hall of Remembrance Item," with "Trivial" connection.